frontend/src/__tests__/integration/pages/modelServing/ModelServingGlobal.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from '@playwright/test';
+
+test('Delete model', async ({ page }) => {
+  await page.goto(
+    './iframe.html?args=&id=tests-integration-pages-modelserving-modelservingglobal--delete-model&viewMode=story',
```

Comment:
What is happening here?

Reply:
it is going to the storybook page without any of the extra controls around it. this procedure is outlined in the testing doc.

Reply:
This needs to be done fair nicer I think... maybe some utility method? I am confused on what exactly is happening in that path -- but it seems boilerplatey and nonsensical to the eye... which screams utility.

Reply:
it kind of is. but you would need to supply the path:

`tests-integration-pages-modelserving-modelservingglobal--delete-model` is coming from `__tests__/integration/pages/modelServing/ModelServingGlobal.tsx` And then the story named `DeleteModel`

in my opinion its more work to pick apart your path for a util function compared to just copy pasting the url from storybook

Reply:
ooooh, I see... interesting. I think we need some utility here based off the path to the file -- but okay, we can leave it for now.